### PR TITLE
Pass param content_only param to overrideLayoutTemplate call

### PR DESCRIPTION
**Pull request by prestamodule**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Pass the content only param to the hook call.
| Type?         | improvement
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| How to test?  | It's just an easy way to get the params "content_only" inside the hook call.

### CODE REVIEW

**Comment by xBorderie** [comment]
I restarted the PHP 7.1 test, just in case.

File: classes/controller/FrontController.php [hunk 1]
```diff
@@ -1321,12 +1323,13 @@ public function getLayout()
                 'entity' => $entity,
                 'locale' => $this->context->language->locale,
                 'controller' => $this,
+                'content_only' => (int) $content_only,
```
Comment (aleeks) [inline classes/controller/FrontController.php]: (int) not needed here

File: classes/controller/FrontController.php [hunk 2]
```diff
@@ -1321,12 +1323,13 @@ public function getLayout()
...
             )
         )) {
             return $overridden_layout;
         }
 
-        if ((int) Tools::getValue('content_only')) {
+        if ((int) $content_only) {
```
Comment (aleeks) [inline classes/controller/FrontController.php]: (int) not needed here

**Review by aleeks — CHANGES_REQUESTED** [review]
Hello,
Just 2 fix to add
Regards

**Review by aleeks — APPROVED** [review]

**Comment by maximebiloe** [comment]
Thank you @prestamodule

**Pull request merged**